src/player.cpp

```diff
@@ -9717,8 +9717,8 @@ const player *player::get_book_reader( const item &book, std::vector<std::string
     const skill_id &skill = type->skill;
     const int skill_level = get_skill_level( skill );
     if( skill && skill_level < type->req && has_identified( book.typeId() ) ) {
-        reasons.push_back( string_format( _( "You need %s %d to understand the jargon!" ),
-                                          skill.obj().name(), type->req ) );
+        reasons.push_back( string_format( _( "%s %d needed to understand. You have: %d" ),
+                                          skill.obj().name(), type->req, get_skill_level( skill ) ) );
```

Comment:
Same here, but `skill_level` is already stashed. I'd also drop the colon from the sentence (here and below).

Reply:
Point taken on `skill_level`, and I wasn't too wedded to the colon.